docs/concepts/publishing.md

```diff
@@ -3,12 +3,9 @@ title: "Publishing"
 description: How to use Flox environments to build artifacts 
 ---
 
---8<-- "paid-feature.md"
-
-Once you've built an artifact with the [`flox build`][builds-concept] command, you likely want to put it somewhere.
-The `flox publish` command gives you the ability to upload artifacts to your private catalog.
+Once you've built an artifact with the [`flox build`][flox-build] command, you likely want to put it somewhere.
+The [`flox publish`][flox-publish] command gives you the ability to upload artifacts to your private catalog.
 In order to share packages with other people you must create an organization.
-This is a paid feature, and if you would like access to it you should contact Flox directly.
 See the [organizations][organizations-concept] page for more details.
```

Comment:
nit wording, IMO the intend is to _use_, more specifically _install_ the package.
For that we need to make it available, i.e. put it into a private catalog.

Reply:
Fixed